install-gn-release/README.md

```diff
@@ -0,0 +1,11 @@
+### Deploy a GeoNetwork War to Maven Repo
+
+Script to deploy GeoNetwork war and contained GeoNetwork artifacts to the IMOS maven repository.
+
+### Usage
+
+./install-gn-release.sh 
```

Comment:
@sachitrajbhandari need to update this to indicate the bucket name parameter

Reply:
Revised Readme file.